docs/components/custom/Markdown/Markdown.js

```diff
@@ -5,20 +5,20 @@ import PropTypes from 'prop-types'
 import { compiler } from 'markdown-to-jsx'
 import mapValues from 'lodash/mapValues'
 // import memoize from 'lodash/memoize';
```

Comment:
This can be removed if not used, also the `eslint-disable` comment at the top of this file.